test/tests.zig

```diff
@@ -1094,6 +1094,7 @@ pub fn addModuleTests(b: *std.Build, options: ModuleTestOptions) *Step {
                     "-std=c99",
                     "-pedantic",
                     "-Werror",
+                    "-Wself-assign",
```

Comment:
> Also...remove self assignments.

Does that mean this line does not need to be added? (related: #19467)

Reply:
I'm enabling a warning, not disabling one.

Reply:
Oops, my mistake!